mirror-main.go

```diff
@@ -174,8 +180,8 @@ func doMirror(sURLs mirrorURLs, progressReader *progressBar, accountingReader *a
 	return sURLs
 }
 
-// doMirrorFake - Perform a fake mirror to update the progress bar appropriately.
-func doMirrorFake(sURLs mirrorURLs, progressReader *progressBar) mirrorURLs {
+// doMirrorFakeProgressUpdate - Update the progress bar to represent a fake mirror
+func doMirrorFakeProgressUpdate(sURLs mirrorURLs, progressReader *progressBar) mirrorURLs {
```

Comment:
umm, is this function name is confusing? I preferred to change it because doMirror() has isFake parameter now so it supports fake mirroring. doMirrorFake would imply the same functionality that's why I decided to change its name